lib/runtime/console.js

```diff
@@ -334,14 +336,16 @@ function shellPty (cwd) {
           cwd = paths.home()
         }
         const env = customEnv()
+        const pty = pty.spawn(atom.config.get("julia-client.consoleOptions.shell"), [], {
```

Comment:
This line will cause undef property error since the global name `pty` is shadowed the local variable.
Need to fix like below:
```suggestion
        const p = pty.spawn(atom.config.get("julia-client.consoleOptions.shell"), [], {
```
and also rename the `pty` in `resolve`